apps/question-the-docs/backend/ai/artifacts.py

```diff
@@ -7,14 +9,17 @@
 
 
 def load_ai_artifacts(db):
-    artifacts = _create_ai_text_artifacts()
-    documents = [Document({settings.vector_embedding_key: v}) for v in artifacts]
-    db.execute(Collection(settings.mongo_collection_name).insert_many(documents))
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
I think we need to reduce the code here (or else move it to another location). This is my reasoning:

1. The idea is illustrate to users what a typical SuperDuperDB app will look like.
2. For this purpose we have created a `components` module and an `artifacts` module. We are suggesting that every SuperDuperDB app should also contain these modules.
3. To explain to users how they can use these modules for other apps, we should try to only put general patterns in these modules. This way, users can understand how these modules can be designed for their own apps.
4. At the moment, we have a bunch of code referring to repos and temporary directories in this module. This code is specific to this app, and might confuse the user ie for a different app, a user does not need to have logic dealing with repos and temporary directories.
5. I believe that we should try to replace this code with a simple abstraction, to make it very clear how to use this module in the future. That is what I tried to do (probably not very well) in the original code.